{
    "type": "software",
    "spec_version": "2.1",
    "id": "software--f2994829-189f-53f4-ab6b-fb5c7c1cea11",
    "name": "IBM Cognos Analytics 11.2.3",
    "cpe": "cpe:2.3:a:ibm:cognos_analytics:11.2.3:*:*:*:*:*:*:*",
    "languages": [
        "en_US"
    ],
    "vendor": "ibm",
    "version": "11.2.3",
    "extensions": {
        "extension-definition--6c453e0f-9895-498f-a273-2e2dda473377": {
            "extension_type": "property-extension",
            "nvd_cpe": {
                "cpe23Uri": "cpe:2.3:a:ibm:cognos_analytics:11.2.3:*:*:*:*:*:*:*",
                "deprecated": false,
                "deprecatedBy": [],
                "lastModifiedDate": "2022-11-18T17:27Z",
                "refs": [
                    {
                        "ref": "https://exchange.xforce.ibmcloud.com/vulnerabilities/233571",
                        "type": "Advisory"
                    }
                ],
                "titles": [
                    {
                        "lang": "en_US",
                        "title": "IBM Cognos Analytics 11.2.3"
                    }
                ],
                "vulnerabilities": [
                    ""
                ]
            }
        }
    }
}